### DISCUSSION TITLE
Request: Tutorials Section for common workflows

### DISCUSSION THREAD

**Discussion by gkamradt** [fuma-nama/fumadocs#892] [Docs Feedback]
In addition to core, ui, and mdx it would be helpful to have a tutorials section that started with a fresh instance of Fumadocs and then implemented a common workflow developers may do.

The current docs do a great job of showing the nuts and bolts, but it is not as intuitive as a hand-holding tutorial.

Two that I ran into:
* Changing the base route from "docs" to "learn" - I run an education based site and want to use fumadocs to host my content. I don't want "docs" as my content route, but rather "learn." I see how to make this edit in the Core > Sources but it's not as clear as I'd like. I spent more time debugging this than I would like
* Adding a route with the same name as your docs route - I like on the Fumadocs site there is home.com/docs which is a landing page for your docs. then there is home.com/docs/ui which is the docs view of your content. When trying to implement the same I ran into "can't have duplicate routes" error. I know the answer is somewhere on the docs if I were to piece it together but it'd be great to have a tutorial on it

If this tutorial section was made I'd be happy to contribute to it

**Comment by fuma-nama** [comment]
Welcome for PRs. I'm busy working on other things so probably isn't able to build and maintain the tutorial section. We do have some guides like navigation and markdown which acts like a tutorial, hope that clarifies it.

About your second point, it's just Next.js layouts using route groups, you can simply consider Fumadocs UI as a component library and you still add layouts like previously in Next.js App Router

**Reply by fuma-nama** [reply]
Also note that the dev branch is in v14, i would suggest to work on the tutorial section after it is officially released if you're interested to contribute.

**Comment by gkamradt** [comment]
Thanks @fuma-nama. I'll wait till v14 comes out then see where energy needs to be put.

Any chance you could point me to the spot in your docs where you handle this routing?

Out of the box I get this error
>  The provided export path '/docs' doesn't match the '/docs/[...slug]' page.

I works fine if I duplicate the layout and put it in app/docs/layout.tsx alongside another page.tsx but I know this isn't right

**Reply by fuma-nama** [reply]
Oops it should be [[...slug]] otherwise it requires one or more segment

**Reply by gkamradt** [reply]
oh interesting, I was trying to mimic what I saw on your docs

How do you manage it here? with just [...slug]
https://github.com/fuma-nama/fumadocs/tree/dev/apps/docs/app/docs

**Reply by gkamradt** [reply]
and fwiw, it runs fine when I npm run dev, works as expected

it just doesn't build and the error throws

**Reply by fuma-nama** [reply]
Because the index page of /docs is in home layout

**Reply by gkamradt** [reply]
I have an index page in there

app/(home)/docs/page.tsx

is that what you mean? Is there another config that needs to be set to tell it to look there for just /docs and then in docs/* for anything else?

appreciate the help, I'm still digging to try and make this work

**Reply by fuma-nama** [reply]
> Is there another config that needs to be set to tell it to look there for just /docs and then in docs/* for anything else?

no, Next.js should handle it properly. Do you have an index.mdx in content/docs? It might be the reason

**Reply by gkamradt** [reply]
Christ almighty, that was it. I had index.mdx in there from testing earlier. Thank you.
